packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -127,7 +127,7 @@ interface AddApiOptions {
 
 const defaultOptions: AddApiOptions = {
   apiName: '\r',
-  testingWithLatestCodebase: true,
+  testingWithLatestCodebase: false,
```

Comment:
? Seems like we want to leave this on.

Reply:
This is set to `true` only for migration tests where we install the older CLI version globally and use the locally built binary as the latest. Unless any migration tests are using these functions without explicitly passing in `testingWithLatestCodebase=true`, we should be good.

Reply:
When it is true, the test consumes the binary installed within the node modules of repo root. In this test the binary installed from local registry will be used and it cannot be consumed by test unless the value is false.